modules/objdetect/include/opencv2/objdetect/aruco_detector.hpp

```diff
@@ -0,0 +1,392 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+#ifndef OPENCV_ARUCO_DETECTOR_HPP
+#define OPENCV_ARUCO_DETECTOR_HPP
+#include <opencv2/objdetect/aruco_board.hpp>
+#include <opencv2/objdetect/aruco_dictionary.hpp>
+
+/** @defgroup aruco ArUco Marker Detection
```

Comment:
Why is it applied to global namespace?

This doesn't work as there is no entry added:
http://pullrequest.opencv.org/buildbot/export/pr/22368/docs/d5/d54/group__objdetect.html

---

We already have some trash for `cv` namespace: https://docs.opencv.org/4.x/namespaces.html

Reply:
move `@defgroup aruco ArUco Marker Detection` to `aruco` namespace